lib/security/allowed_github_ips.json

```diff
@@ -0,0 +1,4999 @@
+{
+  "GitHubAllowIpIpv4": [
```

Comment:
We already have file named with allowed_github_ips.
```suggestion
  "ipv4": [
```

Reply:
Updated.